tests/vm-nesting

```diff
@@ -84,7 +84,12 @@ delete 10
 init 5 --vm
 start 5
 wait 5
-cmd 5 "snap wait system seed.loaded && snap refresh lxd --channel ${LXD_SNAP_CHANNEL:-"latest/edge"}"
+# -minimal images don't have LXD installed
+if [[ "${instanceImage}" =~ -minimal ]]; then
+    cmd 5 "snap wait system seed.loaded && snap install lxd --channel ${LXD_SNAP_CHANNEL:-"latest/edge"}"
```

Comment:
The snap channel env var should skresdt be initialized to s default value so do we need to specify another default here as well?